server/middlewares/menteeChecker.js

```diff
@@ -0,0 +1,16 @@
+import response from '../helpers/response';
+import users from '../models';
+
+module.exports = function(req, res, next) {
```

Comment:
This is `ES5` ensure your whole app is in `ES6`

Reply:
Okay thanks I will Apply the changes to next PR